index.js

```diff
@@ -1,6 +1,19 @@
 /* eslint-env node */
 'use strict';
+let VersionChecker = require('ember-cli-version-checker');
```

Comment:
I think you need to add this to the `package.json`

Reply:
You're right. I misread `ember-cli-dependency-checker` as the version checker.